tests/test_fixerio.py

```diff
@@ -0,0 +1,10 @@
+""" Tests for fixerio module """
+import logging
+from finance_quote import App
+
```

Comment:
the unit test should cover the basic functionality of fixerio (requesting the website).
could you look at the current tests on yahoo and quandl to see practically the kind of test that could be useful ?